### DISCUSSION TITLE
Nested Set

### DISCUSSION THREAD

**Discussion by bandite02** [adonisjs/core#4250] [Japa]
Hi everyone,

AdonisJS is look like Laravel. I'm getting love it.
Is there any package or feature for creating Nested Set like Kalnoy Nested Set in Laravel?. 

Thank you.

**Comment by RomainLanz** [comment]
Hey @bandite02! 👋🏻 

Thanks!

I am not aware of any package to manage nested sets in AdonisJS.
What is your use case?

**Reply by bandite02** [reply]
Hi @RomainLanz 

I'm developing set of Job Position, like CEO, HR Manager, Finance Manager, GA Manager, HR Staff, Finance Staff, etc. Maybe Adonis can create nested set like Kalnoy in laravel.

Thanks.
